CONTRIBUTING.md

```diff
@@ -6,14 +6,12 @@ We'd love for you to contribute to this repository. Before you start, we'd like
     - [Swift style](#swift-style)
     - [Whitespace](#whitespace)
   - [Commits](#commits)
+  - [Before Pushing](#before-pushing)
+
 
 ## Submitting an Issue
-If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue to our repository. Before you submit your issue, search open and closed issues, maybe your question was already answered.
+If you find a bug in the source code or a mistake in the documentation, check our Issues page. It may be that your concern has already been addressed. If you cannot find a satisfying resolution, you can [submit an issue](https://github.com/Pocket/pocket-ios/issues/new/choose).
```

Comment:
curious - do we want a template for our issues? what details should they provide and in what format?

Reply:
That's a good idea!
I just enabled Issues for this repo so when we're setting that up properly we can add some guidance.